### DISCUSSION TITLE
Auto generate llms.txt

### DISCUSSION THREAD

**Discussion by riderx** [withastro/starlight#2617] [Feature Requests]
### What version of `starlight` are you using?

latest

### What is your idea?

Use all the docs file to generate an easy-to-read file for LLM

### Why is this feature necessary?

Public doc should facilitate the work of LLM by publishing content in text only:
https://llmstxt.org/
this will simplify the usage of any product 

### Do you have examples of this feature in other projects?

yes here: https://scalabledeveloper.com/posts/llms-txt-with-astro/

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
I don’t think this needs to be a built-in feature of Starlight. It’s fairly easy to implement manually, and could even be developed as a standalone plugin, so I’d encourage people interested in that to explore that approach.

**Comment by pelikhan** [comment]
Documentation sites will be consumed by humans and LLMs in the near future. It would be nice to be LLM-friendly out of the box. That being said, I have using the https://delucis.github.io/starlight-llms-txt/ which works great for that purpose.

**Comment by HiDeoo** [comment]
Considering that the [`starlight-llms-txt` plugin](https://delucis.github.io/starlight-llms-txt/) exists now, and as Chris [said](https://github.com/withastro/starlight/discussions/2617#discussioncomment-11308688), we don't have plans to implement this in Starlight core so I'll close this discussion for now.
